spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/db/AbstractDatabaseTests.java

```diff
@@ -62,35 +62,18 @@ public void testLatestSharedDb() {
 
 	@Test
 	@DataflowMain
-	public void testLatestSharedDbJdk8() {
+	public void testLatestSharedDbJdk21() {
 		log.info("Running testLatestSharedDb()");
 		// start defined database
 		this.dataflowCluster.startSkipperDatabase(getDatabaseTag());
 		this.dataflowCluster.startDataflowDatabase(getDatabaseTag());
 
 		// start defined skipper server and check it started
-		this.dataflowCluster.startSkipper(TagNames.SKIPPER_main + "-jdk8");
+		this.dataflowCluster.startSkipper(TagNames.SKIPPER_main + "-jdk21");
```

Comment:
Should we add a test for 17 as well as 21?

Reply:
We had tests for 8, 11, 17 so in the same vein I thought we should have 17 and 21.

Reply:
I stand corrected.   I didn't see that we had 17 already.